examples/mqtt_client/Makefile

```diff
@@ -0,0 +1,5 @@
+#!/bin/make
```

Comment:
I don't think you need this shebang line for Makefiles, usually you just run 'make' and it finds the Makefile rather than executing the Makefile.